src/rebar3_hex_user.erl

```diff
@@ -208,7 +208,7 @@ generate_all_keys(Username, Password, LocalPassword, Repo, State) ->
 
     %% write key
     WriteKeyName = api_key_name(),
-    WritePermissions = [#{<<"domain">> => <<"api">>}],
+    WritePermissions = [#{domain => api}],
```

Comment:
This comes from `hex_core`'s API. I'm pretty sure it works in both forms, but, as-is, it's converted later, via `term_to_binary` eventually (if I correctly followed the 🐰 hole).